lib/generators/provider/templates/Rakefile

```diff
@@ -10,6 +9,7 @@ begin
 rescue LoadError
 end
 
+require 'manageiq-providers-<%= provider_name %>'
```

Comment:
Is this even needed?  I think it's automatically required since we don't `:require => false` in the manageiq gemfile